py/torch_tensorrt/fx/converters/aten_ops_converters.py

```diff
@@ -465,6 +502,27 @@ def aten_ops_sym_numel(
     return reduce_layer.get_output(0)
 
 
+@tensorrt_converter(torch.ops.aten.slice.Tensor)
+def aten_ops_slice(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return slice_op(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+        args[2],
+        args[3],
+        args[4],
```

Comment:
This should check if the arguments list is long enough to enable the `step` option. Many models do not specify the `step` and it is omitted from the args list, causing an `IndexError`. Potentially the `or_none` function in this file could be modified slightly to help here:

```python
def arg_bounds_check(args, i, replacement=None):
    return args[i] if len(args) > i else replacement

...

# Replace step
arg_bounds_check(args, 4, replacement=1):
```

Reply:
Ok I will implement this change. Do you have a model that can be run to come across this case?

Reply:
Yes, MobileNetV2, which is tested in the Dynamo E2E test cases elicits this scenario:
https://github.com/pytorch/TensorRT/blob/bd9c29ab046df4605107b8fc530f353c054a5fc3/py/torch_tensorrt/dynamo/test/test_dynamo_backend.py#L50-L78